src/com/lilithsthrone/game/inventory/enchanting/ItemEffectType.java

```diff
@@ -806,7 +806,7 @@ public String applyEffect(TFModifier primaryModifier, TFModifier secondaryModifi
 		
 		@Override
 		public String applyEffect(TFModifier primaryModifier, TFModifier secondaryModifier, TFPotency potency, int limit, GameCharacter user, GameCharacter target, ItemEffectTimer timer) {
-			target.incrementHealth(target.getAttributeValue(Attribute.HEALTH_MAXIMUM)/20);
+			target.incrementHealth(target.getAttributeValue(Attribute.MANA_MAXIMUM)/20);
```

Comment:
incrementHealth, you're sure this is correct?

Reply:
I just check the rest of the file, it should be incrementMana.